aggregates/adapot/src/main/java/com/bloxbean/cardano/yaci/store/adapot/storage/impl/RewardStorageReaderImpl.java

```diff
@@ -1,32 +1,38 @@
 package com.bloxbean.cardano.yaci.store.adapot.storage.impl;
 
-import com.bloxbean.cardano.yaci.store.adapot.domain.InstantReward;
-import com.bloxbean.cardano.yaci.store.adapot.domain.Reward;
-import com.bloxbean.cardano.yaci.store.adapot.domain.RewardRest;
-import com.bloxbean.cardano.yaci.store.adapot.domain.UnclaimedRewardRest;
+import com.bloxbean.cardano.yaci.store.adapot.domain.*;
 import com.bloxbean.cardano.yaci.store.adapot.storage.RewardStorageReader;
 import com.bloxbean.cardano.yaci.store.adapot.storage.impl.mapper.Mapper;
 import com.bloxbean.cardano.yaci.store.adapot.storage.impl.repository.InstantRewardRepository;
 import com.bloxbean.cardano.yaci.store.adapot.storage.impl.repository.RewardRepository;
 import com.bloxbean.cardano.yaci.store.adapot.storage.impl.repository.RewardRestRepository;
 import com.bloxbean.cardano.yaci.store.adapot.storage.impl.repository.UnclaimedRewardRestRepository;
+import com.bloxbean.cardano.yaci.store.api.adapot.dto.RewardInfoType;
 import com.bloxbean.cardano.yaci.store.common.model.Order;
 import com.bloxbean.cardano.yaci.store.events.domain.InstantRewardType;
+import com.bloxbean.cardano.yaci.store.events.domain.RewardRestType;
+import com.bloxbean.cardano.yaci.store.events.domain.RewardType;
 import lombok.RequiredArgsConstructor;
+import org.jooq.DSLContext;
+import org.jooq.impl.DSL;
 import org.springframework.data.domain.PageRequest;
 import org.springframework.data.domain.Pageable;
 import org.springframework.data.domain.Sort;
 
 import java.math.BigInteger;
 import java.util.List;
 
+import static com.bloxbean.cardano.yaci.store.adapot.jooq.Tables.*;
+import static com.bloxbean.cardano.yaci.store.transaction.jooq.Tables.WITHDRAWAL;
```

Comment:
@Sotatek-HuyLe3a Can you please verify if we already have JOOQ WITHDRAWAL class in graalvm native resource file?